reframe/frontend/loader.py

```diff
@@ -86,25 +81,14 @@ def recurse(self):
     def load_from_module(self, module):
         """Load user checks from module.
 
-        This method tries to call the `_get_checks()` method of the user check
-        and validates its return value."""
+        This method tries to call the `_rfm_gettests()` method of the user
+        check and validates its return value."""
         from reframe.core.pipeline import RegressionTest
 
-        old_syntax = hasattr(module, '_get_checks')
```

Comment:
Instead of removing it completely, I could perhaps issue a warning here that this syntax is now dropped and point the users to use the decorators.

Reply:
I think that we can remove completely the old syntax. We should just have a note in the documentation that support for old style checks was removed.

Reply:
I have put a simple check in place that issues a warning, but I have not pushed yet.